src/arviz_stats/psens.py

```diff
@@ -0,0 +1,213 @@
+    It uses Pareto-smoothed importance sampling to avoid refitting the model.
+
+    References
+    ----------
+    .. [1] Kallioinen et al, *Detecting and diagnosing prior and likelihood sensitivity with
+       power-scaling*, 2022, https://arxiv.org/abs/2107.14054
+    """
+    dataset = extract(dt, var_names=var_names, filter_vars=filter_vars, group="posterior")
+    if coords is None:
+        dataset = dataset.sel(coords)
+
+    if group == "log_likelihood":
+        component_draws = get_log_likelihood(dt, var_name=component_var_names, single_var=False)
+    elif group == "log_prior":
+        component_draws = get_log_prior(dt, var_names=component_var_names)
+    else:
+        raise ValueError("Value for `group` argument not recognized")
+
+    component_draws = component_draws.stack(__sample__=("chain", "draw"))
+    if component_coords is None:
+        component_draws = component_draws.sel(component_coords)
+    if isinstance(component_draws, xr.DataArray):
+        component_draws = component_draws.to_dataset()
+    if len(component_draws.dims):
+        component_draws = component_draws.to_stacked_array(
+            "latent-obs_var", sample_dims=("__sample__",)
+        ).sum("latent-obs_var")
+
+    component_draws = component_draws.unstack()
+
+    # calculate lower and upper alpha values
+    lower_alpha = 1 / (1 + delta)
+    upper_alpha = 1 + delta
+
+    # calculate importance sampling weights for lower and upper alpha power-scaling
+    lower_w = np.exp(component_draws.azstats.power_scale_lw(alpha=lower_alpha)).values.flatten()
+    lower_w = lower_w / np.sum(lower_w)
+
+    upper_w = np.exp(component_draws.azstats.power_scale_lw(alpha=upper_alpha)).values.flatten()
+    upper_w = upper_w / np.sum(upper_w)
+
+    return dt.azstats.power_scale_sens(
+        lower_w=lower_w,
+        upper_w=upper_w,
+        delta=delta,
+    )
+
+
+def psens_summary(data, threshold=0.05, round_to=3):
+    """
+    Compute the prior/likelihood sensitivity based on power-scaling perturbations.
+
+    Parameters
+    ----------
+    data : DataTree
+    threshold : float, optional
+        Threshold value to determine the sensitivity diagnosis. Default is 0.05.
+    round_to : int, optional
+        Number of decimal places to round the sensitivity values. Default is 3.
+
+    Returns
+    -------
+    psens_df : DataFrame
+        DataFrame containing the prior and likelihood sensitivity values for each variable
+        in the data. And a diagnosis column with the following values:
+        - "prior-data conflict" if both prior and likelihood sensitivity are above threshold
+        - "strong prior / weak likelihood" if the prior sensitivity is above threshold
+        and the likelihood sensitivity is below the threshold
+        - "-" otherwise
+    """
+    pssdp = psens(data, group="log_prior")["posterior"].to_dataset()
+    pssdl = psens(data, group="log_likelihood")["posterior"].to_dataset()
```

Comment:
```suggestion
    pssdp = psens(data, group="log_prior")
    pssdl = psens(data, group="log_likelihood")
```

with the change above on the outpur of `psens`